src/typings/global.d.ts

```diff
@@ -54,6 +54,8 @@ declare global {
     };
     MathJax: any;
     ga: UniversalAnalytics.ga;
+    dataLayer:  object[];
+    gtag: any;
```

Comment:
are there types in `UniversalAnalytics` to help with this? if not then you could at least type gtag as a function accepting `string, string, object` which i think is what we understand to be the right arguments